rskj-core/src/main/java/org/ethereum/vm/PrecompiledContracts.java

```diff
@@ -121,8 +120,19 @@ public PrecompiledContract getContractForAddress(BlockchainConfig blockchainConf
         if (address.equals(REMASC_ADDR_DW)) {
             return new RemascContract(config, new RemascConfigFactory(RemascContract.REMASC_CONFIG).createRemascConfig(config.netName()), REMASC_ADDR);
         }
-        if (blockchainConfig.isRskip106() && address.equals(BTOUTILS_ADDR_DW)) {
-            return new BTOUtils(config, BTOUTILS_ADDR);
+
+        // Precompiled contracts depending on blockchainConfig
+        if (blockchainConfig != null) {
```

Comment:
How could we allow this to be `null`?!?!?

Reply:
It is null in some unit tests.

Reply:
Is it reasonable to fix them as part of this PR?

Reply:
This is not mandatory but it will be nice to follow @tinchou comment

Reply:
I think I can handle this in a future refactor, but leaving a comment would be nice.